### DISCUSSION TITLE
Tips for integrating with react framework requiring postcss?

### DISCUSSION THREAD

**Discussion by kyczawon** [silviogutierrez/reactivated#333] [General]
I am using [Mantine](https://mantine.dev/getting-started/) for my reactivated project, and the most recent Mantine version v7 changed to rely on [postcss](https://postcss.org/). So I need to integrate postcss into my build step to be able to migrate to the last version.

They recommend using webpack and adding a 'postcss-loader' (see [here](https://mantine.dev/getting-started/)), which can then generate relevant files based on any css imported.

1. Could postcss be integrated into the reactivated build step? Is there anyway I could set up reactivated to deal with this for me?
2. Should I integrate webpack into my project instead?

I appreciate any tips on this :)

**Comment by silviogutierrez** [comment]
See #326 . Hope that helps.

**Reply by kyczawon** [reply]
Thanks, this is amazing! Sorry for the duplicate - I searched the discussions, but not the issues.
